src/main/java/com/scottlogic/Main.java

```diff
@@ -12,7 +12,7 @@ public static void main(String[] args) {
                 OffsetDateTime.of(2020, 1, 3, 7, 12, 3, 0, ZoneOffset.UTC),
                 "Hello World!", 2);
 
-        UserPost userPost2 = new UserPost("Joe Bloggs",
+        UserPost userPost2 = new UserPost("Matt Damon",
                 OffsetDateTime.of(2020, 1, 3, 8, 53, 34, 0, ZoneOffset.UTC),
                 "Another example post.", 1);
```

Comment:
We've allowed you to add data like this so far but it is quite messy for the class.
Could you create a 'utils' directory and move these placeholder data there as separate classes.
For example, UserPostData.java for the user posts.

We'll go ahead with this until they can be removed if you link to a DB.